src/AppBundle/Converter/TwigTemplateConverter.php

```diff
@@ -113,6 +113,12 @@ public function convertLayoutTemplate($mjmlTemplatePath, $mjmlTheme, $newTheme)
         $head->insertBefore($blockStyleStart, $style);
         $head->appendChild($blockStyleEnd);
 
+        //Add inline CSS for RTL languages
+        $link = $dom->createTextNode("{% if languageIsRTL %}\n  ".
+            "<style type=\"text/css\">{% include '@MailThemes/modern/assets/rtl.css' %}</style>\n  ".
+            "{% endif %}\n  ");
```

Comment:
The whitespaces on the first line is here for the indentation I guess, but the other ones are not needed (unless I'm missing something ^^).
```suggestion
            "<style type=\"text/css\">{% include '@MailThemes/modern/assets/rtl.css' %}</style>\n".
            "{% endif %}\n");
```